manifests/h/Helix/Helix/22.08.1/Helix.Helix.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Helix.Helix
+PackageVersion: 22.08.1
+InstallerType: zip
+Commands:
+- hx
+ReleaseDate: 2022-09-01
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VC++2015-2019Redist-x64
  - PackageIdentifier: Microsoft.VC++2015-2019Redist-x86
```
Would this work perhaps? Found this [here](https://github.com/microsoft/winget-pkgs/blob/92b8d5d7727aa641aa3d758dbd653a4c8ba4ec8c/manifests/v/VMware/WorkstationPro/16.2.3/VMware.WorkstationPro.installer.yaml#L20-L23).

---

Unfortunately, that's the old name for the same package, alongside the x86 version. Winget currently only installs the first dependency, so I believe stacking them in this way allows Winget to install the x86 version on 32-bit OSes, where the x64 version is inapplicable. Since this is an x64 package, only the x64 dependency should be necessary.